modules/ui/commit_changes.js

```diff
@@ -85,6 +93,36 @@ export function uiCommitChanges(context) {
             .on('click', zoomToEntity);
 
 
+        // Download changeset link
+        var changeset = new osmChangeset({ id: 'CHANGEME' }),
```

Comment:
Generally for downloading you don't want to supply anything for changeset ID.

Reply:
Can you check this for me?  I'm really not sure what other tools need.

It currently ends up generating something kind of like this, but we can remove the `changeset="CHANGEME"` stuff if needed:
```xml
<osmChange version="0.6" generator="iD">
  <create/>
  <modify>
    <node id="105439801" lon="-74.53666020842094" lat="40.663296528227306" version="2" changeset="CHANGEME"/>
    <node id="105439798" lon="-74.53607265426908" lat="40.66309431604051" version="2" changeset="CHANGEME"/>
  </modify>
  <delete if-unused="true">
    <node id="5106517631" lon="-74.5367492" lat="40.6633484" version="1" changeset="CHANGEME"/>
  </delete>
</osmChange>
```

Reply:
Remove the changeset="CHANGEME" and it looks okay. I'm not 100% sure what programs support reading osmChange for uploads.

Reply:
sounds good, thanks!